docs/topics/jvm/jvm-test-using-junit.md

```diff
@@ -1,20 +1,17 @@
 [//]: # (title: Test code using JUnit in JVM – tutorial)
 
-This tutorial will show you how to write a simple unit test and run it with the Gradle build tool.
+This tutorial shows you how to write a simple unit test and run it with the Gradle build tool.
 
-The example in the tutorial has the [kotlin.test](https://kotlinlang.org/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.
+The example in the tutorial has the [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.
 
 To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
 
 ## Add dependencies
 
-1. Open a Kotlin project in IntelliJ IDEA. If you don't already have a project, [create one](jvm-get-started.md#create-an-application).
+1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project, [create one](jvm-get-started.md#create-an-application).
```

Comment:
```suggestion
1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project, [create one](jvm-get-started.md#create-a-project).
```

Reply:
The instructions linked here tell the user to use IntelliJ for the build system, not Gradle. So maybe we shouldn't link to these steps? 🤔

Reply:
Yeah, you're right.
I changed the link to the IDEA's tutorial.